catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -347,12 +347,20 @@ public void init() {
                     .map(Result::getMetacard)
                     .map(metacard -> {
                         boolean isSubscribed = ids.contains(metacard.getId());
-
-                        return ImmutableMap.builder()
-                                .putAll(transformer.transform(metacard))
-                                .put("subscribed", isSubscribed)
-                                .build();
+                        try {
+                            return ImmutableMap.builder()
+                                    .putAll(transformer.transform(metacard))
+                                    .put("subscribed", isSubscribed)
+                                    .build();
+                        } catch (RuntimeException e) {
+                            LOGGER.warn(
+                                    "Could not transform metacard. WARNING: This indicates there is invalid data in the system. This is caused by invalid or malformed data being manually inserted into the catalog. Please advise your administrator to check that all 3rd party integrations are uploading valid data. [Workspace Metacard - queries field]  Metacard: title='{}', id='{}'",
```

Comment:
This log message shouldn't make any assumptions about how the data got into the system or who did it.  It is possible it was put into the system by an integrator, but that isn't an appropriate assumption to put here.  Please update this log message to simply state that the data is invalid along with the metacard info.  Also, I think there should be instructions on how an administrator can remove the offending metacard.